basic-steps/src/main/resources/org/jenkinsci/plugins/workflow/steps/TimeoutStep/help.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Executes the code inside the block with a determined time out limit.
+    If the time limit is reached, the build is aborted.
```

Comment:
Technically, an exception is thrown which if uncaught will result in the build being aborted. Not sure if that is important enough to explain here, though.